ext/broadcast_channel/01_broadcast_channel.js

```diff
@@ -98,7 +98,7 @@
     }
 
     postMessage(message) {
-      webidl.assertBranded(this, BroadcastChannel);
+      webidl.assertBranded(this, BroadcastChannel.prototype);
```

Comment:
This actually introduces a new issue: users could override `BroadcastChannel.prototype`. We need to "primordialize" this as `BroadcastChannelPrototype` I think.

Reply:
Makes sense, given that this is our internal code we will need to provide two "exports" for each of this classes, right?

Reply:
Yeah, probably.